### ISSUE TITLE
Move icons further apart

### ISSUE THREAD

**Issue by balajiramanathan** [pawelsalawa/sqlitestudio#4497]
### Details
I would like the icons for adding rows and deleting rows to be moved farther apart (maybe to the other end of the window), and similarly, move apart the icons for committing and rolling back entered data.  I find that it is very easy to hit the wrong icon because they are close to each other and the consequences of hitting one versus the other are so radically different.

Also, there is no confirmation for deletion or rolling back, so if I do hit the wrong button by mistake, there is no way to recover from it.  And there is no undo either, so I have to start all over again and repair the database (if I have deleted rows I should not have), and then go back to whatever editing I wanted to do.

![image](https://user-images.githubusercontent.com/11827185/187039281-9de1db6e-40c3-4959-9ddb-24b42ca7a504.png)

My suggestion is to move the yellow-highlighted icons to the red rectangle area.  Thank you.

### Steps to reproduce
This applies only to bug reports. If this is a feature request, delete this section.

### Operating system
This applies only to bug reports. If this is a feature request, delete this section.

### SQLiteStudio version
This applies only to bug reports. If this is a feature request, delete this section.

**Comment by pawelsalawa** [comment]
I'd say that making it configurable would be more suitable for all users, because if it was just moved forever, than other users will come complaining.

I will rename title to reflect that. I hope you don't mind.

**Comment by balajiramanathan** [comment]
No, not at all.  Yes, having the ability to move those icons (or even remove the "dangerous" ones and have them accessible only via a dropdown) is fine with me.  Just like MS Office allows you to choose what appears on the toolbar, and where.
